ui/shared/AddressRequestedValidator.qml

```diff
@@ -0,0 +1,40 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import "../imports"
+import "./status"
+
+Column {
+    id: root
+    anchors.horizontalCenter: parent.horizontalCenter
+    spacing: 5
+
+    visible: !isValid || isWarn
+
+    property bool isValid: true
+    property bool isTxRequest: false
+    property bool isWarn: !isTxRequest
```

Comment:
Is this reversed? We wanted to show a warning when it is a tx request, correct?

Also, we could consider checking an address value passed in for 0x0, which would cater for the ENS case.

Reply:
>  We wanted to show a warning when it is a tx request, correct?
We want to show the warning when we **send** a request. Every time a tx should be sent, the receiver needs to share a wallet address first. 
<img width="469" alt="Screenshot 2020-10-26 at 11 24 52" src="https://user-images.githubusercontent.com/445106/97161274-e0838a80-177d-11eb-81bb-75d507dedacd.png">